inference/core/utils/preprocess.py

```diff
@@ -63,7 +63,14 @@ def auto_orient(image):
     return image
 
 
-def prepare(image, preproc):
+def prepare(
+    image,
+    preproc,
+    disable_preproc_auto_orient: bool = False,
+    disable_preproc_contrast: bool = False,
+    disable_preproc_grayscale: bool = False,
+    disable_preproc_static_crop: bool = False,
```

Comment:
what is the PRO of doing it this way rather than another dict of keys to disable, that have to match `preproc` keys?

Reply:
Ah, is this due to how we are getting these on the request?

Reply:
> Ah, is this due to how we are getting these on the request?

Ya, it would be possible to do it either way, with the API updates we moved away from object-like args completely in favor of keyword args, but maybe there's a happy medium. I think we can take another stab at the API for a 1.0 release in the future.